rules/windows/process_creation/proc_creation_win_control_panel_item.yml

```diff
@@ -30,6 +30,7 @@ detection:
         CommandLine|contains:
             - '\System32\'
             - '%System%'
+            - '|C:\Windows\system32|'
```

Comment:
Is the pipe intentional?

Reply:
Confirmed it is because windows is fun 🗡️

Reply:
Fro reference the CLI looks like `taskhost.exe 00000039|<user>|C:\Windows\system32|appwiz.cpl`